phoenix-core/src/it/java/org/apache/phoenix/end2end/ImmutableTableIT.java

```diff
@@ -98,4 +99,13 @@ public void testQueryWithMultipleColumnFamiliesAndMultipleConditionsForImmutable
             }
         }
     }
+
+    @Test
+    public void testImmutableTableCreation() throws Exception {
```

Comment:
We already have immutable table creation in other tests in this class. Do we need this test?

Reply:
Correct, @ChinmaySKulkarni , doesn't look like we need this. Removed it.